### ISSUE TITLE
KeyAction does nothing 

### ISSUE THREAD

**Issue by habhabhabs** [pywinauto/pywinauto#662]
Based on issue #503, v0.6.5 should be able to simulate holding a key until it is programmatically released. 

Do correct me if I'm wrong on the operation of KeyAction, otherwise this is not working. Thanks!

Cheers

**Comment by vasily-v-ryabov** [comment]
This is just implemented in master branch and should be included into 0.6.6. You can check it with master branch by
```
pip uninstall pywinauto
pip install https://github.com/pywinauto/pywinauto/archive/master.zip
```
Sorry for long waiting for 0.6.6. There are very few issues left: [milestone "pywinauto 0.6.6"](https://github.com/pywinauto/pywinauto/milestone/14) but it may take 2 weeks since I work on the project out of office and family hours. Also replying to all questions takes some time.

**Comment by habhabhabs** [comment]
Thanks @vasily-v-ryabov! That was prompt and appreciate your team's hard work.
